src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -83,6 +84,12 @@
     @Column
     private boolean browsable;
 
+    @OneToMany(mappedBy = "study", cascade = CascadeType.ALL)
+    private List<LinkedStudy> linkedStudies;
+
+    @OneToMany(mappedBy = "linkedStudy", cascade = CascadeType.ALL)
+    private List<LinkedStudy> linkedStudiesOf;
```

Comment:
Of? can you explain the difference between linked studies and linkedStudiesOf?

Reply:
It is same idea as in https://stackoverflow.com/a/1656261. The author explained better than me.